canvas_modules/harness/cypress/integration/properties/common-properties-actions.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017-2020 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe("Test of common properties actions", function() {
+	before(() => {
+		cy.visit("/");
+		cy.openPropertyDefinition("action_paramDef.json");
+	});
+
+	it("Test of Increment, Decrement actions", function() {
+		cy.clickAction("Increment");
+		cy.clickAction("Increment");
+		cy.clickAction("Increment");
+		cy.verifyReadOnlyNumericValue(3);
+		cy.clickAction("Decrement");
+		cy.verifyReadOnlyNumericValue(2);
+	});
+});
+
+describe("Test of text styling and word wrapping", function() {
```

Comment:
I would put this in another file for readonly control tests.  Same with the other test case as well.